monkey/monkey_island/cc/resources/ransomware_report.py

```diff
@@ -0,0 +1,10 @@
+import flask_restful
+from flask import jsonify
+
+from monkey_island.cc.resources.auth.auth import jwt_required
+
+
+class RansomwareReport(flask_restful.Resource):
+    @jwt_required
+    def get(self):
+        return jsonify({"show": True, "report": None})
```

Comment:
The back end should not be making decisions about what the front end should show. It's the backend's job to report what data it has. It's the frontend's job to decide how to display that data.